src/levanter/infra/ray_tpu.py

```diff
@@ -583,6 +589,16 @@ def _make_unique_job_id(client, run_id):
     return job_id
 
 
+def _deep_merge_envs(env1, env2):
```

Comment:
This is not called?

Reply:
oh sorry, i had pycharm inline it but it kept the declaration